docker/README.md

```diff
@@ -0,0 +1,61 @@
+# ION using Docker 
+
+This directory contains the automated Docker deployment of ION.  This will create 5 docker containers that contain the core components to run ION.
+
+- bitcoin-core - This container will run the bitcoin-core client.  `NOTE: This is currenlty using testnet as the source chain`
+
+- bitcoin-ion - This is the bitcoin sidetree node for ION.
+
+- ipfs - This is the local IPFS node used by the sidetree.
+
+- mongo - This is the local MongoDB used by the sidetree.
+
+- core-ion - This is the core sidetree node.
+
+The automation will create the docker containers in order (and will wait for bitcoin sync).
+
+## Prerequisites
+
+This implementation is designed to be run with very little dependencies, on a Linux host.
+
+- Linux Ubuntu 18.04LTS  `NOTE: You will need about 31 GB of free space for bitcoin testnet.`
+
+- Docker engine (latest)
+  ```
+  sudo apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+
+  curl -fsSL --max-time 10 --retry 3 --retry-delay 3 --retry-max-time 60 https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+  sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+
+  sudo apt-get update
+  sudo apt-get install -y docker-ce
+  sudo systemctl enable docker
+  ```
+
+- Docker compose (latest)
+  ```
+  sudo curl -L --max-time 10 --retry 3 --retry-delay 3 --retry-max-time 60 "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+
+  sudo chmod +x /usr/local/bin/docker-compose
+  ```
+
+- NodeJS - version 10.x   `NOTE: This has a requirement for version 10 of nodejs only`
```

Comment:
What is the cause for requiring specifically Node v10.x?

Reply:
No hard requirement from me, but it was stated in the install guide [here](https://github.com/decentralized-identity/ion/blob/master/install-guide.md) to use NodeJS v10.  I have not tested with 12, but would be good to support this.

Reply:
@thehenrytsai sounds like we need to update the guide to list v12? Are there any other things to change in the guide along with this?

Reply:
@csuwildcat, yes, we should change to v12. I don't know if there are other changes required as I have not gone through the installation guide for a while, but we do try to update them when we make changes that we know impact the installation process..

Reply:
@thehenrytsai  any update, ok to use v12 nodejs?